feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -374,10 +382,19 @@ func TestTrafficWithGracefulRestartSpeaker(t *testing.T) {
 		t.Log("Start DUT interface Config")
 		configureDUT(t, dut)
```

Comment:
**configureDUT()** function configures DUT related objects. So we can move the NetworkInstance configuration to this function.

Reply:
Moved NI configurations under configureDUT(). Thank you